minimal-kernel/README.md

```diff
@@ -0,0 +1,24 @@
+# Kernel Gateway Stack
+
+## What it Gives You
+* [Jupyter Kernel Gateway](https://github.com/jupyter-incubator/kernel_gateway) that spawns minimal Jupyter Python kernel
+* No preinstalled scientific computing packages
+* Unprivileged user `jovyan` (uid=1000, configurable, see options) in group `users` (gid=100) with ownership over `/home/jovyan`
+* [tini](https://github.com/krallin/tini) as the container entrypoint
+* Options for Port and Ip address
+
+
+## Basic Use
+
+The following command starts a container with the Kernel Gateway server listening for HTTP connections on port 8888.
```

Comment:
Maybe state that it runs as `nobody` so that people know it's reasonably secure by default.